docs/source/intervention_models/targeted_bep_and_mms/index.rst

```diff
@@ -95,35 +95,59 @@ IFA is widely used as a prenatal supplement in most areas of the world and is re
 Baseline Coverage Data
 ++++++++++++++++++++++++
 
-.. todo::
-
-  Document known baseline coverage data, using the table below if appropriate
+Given the low utilization of MMS and BEP relative to IFA, we assume that baseline coverage of MMS and BEP are zero. Baseline coverage of IFA varies by location, and Demographic Health Surveys are good data sources for the proportion of pregnant women who took iron supplementation during pregnancy. 
 
 .. list-table:: Baseline coverage data
-  :widths: 15 15 15 15 15
   :header-rows: 1
 
   * - Location
     - Subpopulation
     - Coverage parameter
     - Value
     - Note
-  * - 
-    - 
-    - 
-    - 
-    - 
+  * - Ethiopia
+    - General population
```

Comment:
If this is among the general population, should we adjust it up to apply to the pregnant population?

Reply:
Yes, good point! Just made that change.